transforms/transforms.go

```diff
@@ -61,6 +62,9 @@ func Fieldmap(mappings map[string][]string) optimus.TransformFunc {
 		for key, vals := range mappings {
 			for _, val := range vals {
 				if oldRowVal, ok := row[key]; ok {
+					if _, ok := newRow[val]; ok {
+						return nil, fmt.Errorf("Multiple Fieldmap exists for %s", val)
```

Comment:
Can we clarify this error message? I don't think it's obvious what "Multiple Fieldmap" means. Maybe something like "Detected multiple mappings to the same value for key %s"